src-vue/src/views/RepairView.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <div class="fc-container">
```

Comment:
This should have no padding-top, since there's no menu on this view.

Reply:
Fixed in e7c6746a17a8ec0d9fa770be24cf0c0940330343